api/v1alpha1/ext_authz_type.go

```diff
@@ -0,0 +1,12 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+// ExtAuthz defines the configuration for External Authorization.
+type ExtAuthz struct {
+	// GRPCURI defines the gRPC cluster name to use in that route.
```

Comment:
we probably need a `type` field in here to distinguish between http and grpc authz server so the config would look like
```
type: "GRPC"
grpc:
```